main.go

```diff
@@ -34,6 +37,13 @@ const APIOffsetAddr uintptr = 0x33EE420 // 1.09
 const GGStriveAPIURL = "https://ggst-game.guiltygear.com/api/"
 const PatchedAPIURL = "http://127.0.0.1:21611/api/"
 
+const GithubDownloadURL = "https://github.com/optix2000/totsugeki/releases/latest/download/"
+const GithubReleasesURL = "https://api.github.com/repos/optix2000/totsugeki/releases/latest"
+
+const UpdateTimeout = 5 * time.Second
```

Comment:
5 Seconds is a bit too short for people with slow internet. 30 seconds seems pretty reasonable (for now).